src/steamship/invocable/mixins/blockifier_mixin.py

```diff
@@ -19,6 +20,7 @@ def blockify(
         file_id: str,
         mime_type: Optional[MimeTypes] = None,
         blockifier_handle: Optional[str] = None,
+        after_task_id: Optional[str] = None,
     ) -> Task:
         """Blockify the file `file_id` using a curated set of defaults for the provided `mime_type`.
```

Comment:
This feels like a very thin wrapper around the normal call to file.blockify.  Does this content just belong in the main SDK rather than as a mixin?  Is there value in exposing the _route_ `blockify_file`, or is this just a step in the larger process?

Reply:
I got the read folks were a bit hesitant to have a `auto-blockify` function in the SDK proper since it has specific knowledge of plugins we've deployed.. whereas here it feels somehow more OK to have specific knowledge of plugins we've deployed.

Also: it feels easier to put things like the update-file-status here than the SDK, since they're really UI niceties that aren't actually needed (until we decide to officiate them) at the sdk level.

I'm agnostic/unsure